### PR TITLE
fix: In member clusters below v3.3.0, some functions should be disabled and prompt to upgrade

### PR DESCRIPTION
**Pull request by weili520**
Signed-off-by: TheYoungManLi <cjl@kubesphere.io>
### What type of PR is this?
/kind bug

### Which issue(s) this PR fixes:
Fixes ##3335,##3336

### Special notes for reviewers:
```
```

### Does this PR introduced a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: src/pages/clusters/containers/Storage/StorageClasses/Detail/index.jsx [hunk 1]
```diff
@@ -50,9 +50,11 @@ export default class StorageClassDetail extends React.Component {
     shouldAddCrd: false,
   }
 
-  componentDidMount() {
+  async componentDidMount() {
+    const { params } = this.props.match
     this.store.fetchList({ limit: -1 })
     this.fetchData()
+    this.ksVersion = await this.store.getKsVersion(params)
```
Comment (harrisonliu5) [inline src/pages/clusters/containers/Storage/StorageClasses/Detail/index.jsx:57]: the `fetchData` function has got the value of ksVersion.

File: src/pages/projects/containers/Volumes/Detail/index.jsx [hunk 2]
```diff
@@ -40,8 +40,10 @@ export default class VolumeDetail extends React.Component {
 
   storageclass = new StorageClass()
 
-  componentDidMount() {
+  async componentDidMount() {
+    const { params } = this.props.match
     this.fetchData()
+    this.ksVersion = await this.store.getKsVersion(params)
```
Comment (harrisonliu5) [inline src/pages/projects/containers/Volumes/Detail/index.jsx:46]: maybe in fetchData to get ksVersion is better.

Comment (weili520) [reply src/pages/projects/containers/Volumes/Detail/index.jsx]: updated

File: src/pages/clusters/containers/Storage/StorageClasses/Detail/index.jsx (hunk 1 above)
Comment (weili520) [reply src/pages/clusters/containers/Storage/StorageClasses/Detail/index.jsx]: updated

**Pull request merged**